inst/validation/bbr-requirements.yaml

```diff
@@ -619,13 +619,9 @@ PEST-R001:
   tests:
   - BBR-PEST-001
 PEST-R002:
-  description: param_estimates correctly errors on Bayesian model
+  description: param_estimates correctly errors on: Bayesian model, Bayesian model with multiple estimation methods, ONLYSIM models
```

Comment:
I guess the common factor here is just there they're some sort of unsupported model, so what do you think about "param_estimates errors on unsupported models"?

Reply:
Yeah I thought about that. Part of me likes the idea of referencing the specific tests they correspond to, mainly in the event we added another unsupported model that deserved its own requirement for some reason. Dont have a strong opinion though and am ok with that change

Reply:
Hmm, yeah, either way is fine with me.  I find the current form a bit wordy, and it seems written in a way that's more likely to need to be updated for new tests, but that's not a big concern.  Whatever you want to go with woks for me.